File: JS, PHP, PYTHON/JAVASCRIPT/FOR.js
function mostrarPalabra() {
    let palabra = prompt("Ingrese la palabra que quiere que se repita:");
    
    // For para imprimir la palabra 10 veces
    for (let i = 0; i < 10; i++) {
        console.log(palabra);
    }
}

mostrarPalabra();

File: C++ y Java/C++/For.cpp
#include <iostream>
#include <string>

int main()
{
  std::string palabra;

  std::cout << "Ingrese la palabra que quiere que se repita: ";
  std::getline(std::cin, palabra);

  for (int i = 0; i < 10; i++)
  {
    std::cout << palabra << std::endl;
  }

  return 0;
}